public/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>eBay Product Page Clone</title>
+  </head>
+  <body>
+    <div id="Gallery"></div>
+    <div id="product-description"></div>
+    <!-- <div id="Related"></div> -->
+    <div id="reviews"></div>
+    <script src="/lib/react.development.js"></script>
+    <script src="/lib/react-dom.development.js"></script>
+    <!-- <script type="text/javascript" src="http://localhost:3001/app.js"></script> -->
+    <script type="text/javascript" src="http://localhost:3005/bundle.js"></script>
```

Comment:
Looks good, just a heads up when you update my service code the port is updated to 3002